### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by AndehX** [ho0ber/NK2Tray#109]
Doesn't seem to be any active development going on judging by the lack of updates to the github...

**Comment by ho0ber** [comment]
Yeah, the previously active developers have had to step away and I'm not really set up to do development on native windows apps.

I'm currently working on a rewrite that, strictly speaking, is a less ideal choice of language, but will be a lot easier for me to continue to be personally involved, push bug fixes, and add device support.

There are some great alternative projects out there, though most of them are not open source.

More details on similar projects and on the rewrite can be found on the discord in #rewrite-progress and #similar-projects:
https://discord.gg/2NtJnZPVRT

**Comment by AndehX** [comment]
Very interesting, thanks!  Honestly, even though development on this seems to have stopped, i'm just glad to hear that you're working on a rewrite, since the current version has some issues (namely it seems to randomly "forget" programs that i've assigned, after a reboot)

**Comment by soepketel-dev** [comment]
Im using https://www.midi-mixer.com/ as alternative
